index.html

```diff
@@ -176,6 +230,12 @@ <h4>unstable-webxr</h4>
         <h3 class="grouping-title" style="margin-top: 40px;">EVENTS</h3>
 
         <ul class="list-attribute">
+          <li>
+            <h4>environment-changed</h4>
```

Comment:
Not sure about exposing/publicizing this (yet, at least) -- as this will fire for default environment maps created, and fire on initial load, even if no background provided (default env), as well for every model load -- I haven't thought of this as a public API, anyway -- if so, we should use present tense e.g. `environment-change`

Reply:
Every event fired by an element is public API whether you advertise it or not, and this one has been in the element for some time now.

Will change the tense, although otherwise your description sounds like what this event was intended to do.

Reply:
sgtm :+1: just pointing it out since I've not personally thought about this as a public API (just for tests), no objection if you've thought about the ramifications. I don't recall the reasoning why this event is on the element as opposed to something private, like the model (e.g. model already has an `envmap-change` event)

Reply:
Its primary use today is for tests, but given that there is an ephemeral period where it is unclear if the environment map has changed after setting `background-image` or `environment-image`, this seems like something that will be indispensable for many use cases. Without it, users will almost always have to detail with FOUC.

Eventually when we add progress bars and related events, it might not make as much sense to have such a specific event. But, right now it seems really important.